include/boost/url/decode_view.hpp

```diff
@@ -1001,6 +1005,24 @@ operator<<(
     std::ostream& os,
     decode_view const& s);
 
+/** Return the string as a range of decoded characters
+
+    @par Complexity
+    Constant.
+
+    @par Exception Safety
+    Throws nothing.
+
+    @see
+        @ref decode_view.
+*/
+inline
+decode_view
+operator*(pct_string_view s) noexcept
```

Comment:
IF you do this then the operator will just be by itself in the docs and not attached to `pct_string_view` where it should be...